### DISCUSSION TITLE
Nouveau simulateur ACRE

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2623] [Nouveaux simulateurs]
D'après un retour utilisateur : 

> Si je dépasse les plafonds, est ce que l'aide s'applique pour les premiers paliers, puis ensuite plus pour le reste de ma rémunération ? Ou est ce que dépassé 40000, j'aurai aucune aide du tout ?

> Il semble que sur des sites, j'ai l'impression de lire que l'aide ne s'applique pas du tout, sur d'autres site l'info n'est pas claire, mon expert comptable pense que l'aide s'applique pour les 30000 premiers euros mais n'est pas sur.

> Bref, un simulateur d'exonération ACRE aurait pu répondre à la question, je devrais pouvoir entrer ma rémunération, dates de début d'activité... et le simulateur affiche les cotisations et l'éxonération.

**Comment by mquandalle** [comment]
On a eu plusieurs retours sur ce sujet et donc ça peut être intéressant à traiter. Avant de se lancer dans un simulateur dédié on pourrait mieux informer sur le montant de l'ACRE et la manière dont il est calculé.

- ajouter un bloc bleu à la manière de ce que l'on fait sur la régularisation progressive ? 
![image](https://user-images.githubusercontent.com/1730702/132373516-7f5e165f-f929-4c13-94af-ad4ad0a49aa2.png)

- Ajouter une ligne "dont réduction ACRE" dans le bloc bleu ?
![image](https://user-images.githubusercontent.com/1730702/132373618-34d47994-8da0-4a03-a963-f8c7ebfc0596.png)
